Kudu.Client/Kudu.Client.csproj

```diff
@@ -7,6 +7,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>Kudu.Client</RootNamespace>
     <AssemblyName>Kudu.Client</AssemblyName>
+    <NuGetPackageImportStamp>97469353</NuGetPackageImportStamp>
```

Comment:
No need to commit this (or others like it). See http://stackoverflow.com/questions/24049961/what-is-nugetpackageimportstamp-for